packages/react-core/src/components/Chip/Chip.tsx

```diff
@@ -38,6 +38,9 @@ export interface ChipProps extends React.HTMLProps<HTMLDivElement>, OUIAProps {
     | 'left-end'
     | 'right-start'
     | 'right-end';
+
+  /** Css property expressed in percentage or every css unit that override the default value of the max-width  */
```

Comment:
I would updated to this 
```suggestion
  /** Css property expressed in percentage or any css unit that overrides the default value of the max-width of the chip's text  */
```